hyperlogic/hifi
# 7429 IPD Secondary Task List: bubble.js Scale

7429 IPD Secondary Task List: bubble.js Scale

https://github.com/Delanir/hifi/commit/4a1dd975a92ef3bd8723db0f4566db86d5040494

Test Plan:
- Find someone who can help you activate the bubble.
- Turn on the Bubble
-  Increase your avatar size (double size is ok). 
- Walk up to your helper.

Expected: The Bubble will trigger and will scale up with your avatar when you reach your helper.

- Find someone who can help you activate the bubble.
- Turn on the Bubble
-  Make yourself tiny (think ant-man is ok). 
- Walk up to your helper.

Expected: The Bubble will trigger and will scale down with your avatar when you reach your helper.
